src/script_parser.rs

```diff
@@ -0,0 +1,178 @@
+use crate::characters::Character;
+use regex::Regex;
+use std::collections::HashMap;
+
+/// Identifies the title in the script text
+/// # Errors
+/// Returns 452 to be used as a status code that the frontend interprets as unable to find a title
```

Comment:
I'd suggest separating the "frontend error code" stuff from the parsing code and possibly return something more descriptive than a number.